test/system/helpers.bash

```diff
@@ -428,6 +428,18 @@ function skip_if_cgroupsv1() {
     fi
 }
 
+######################
+#  skip_if_rootless_cgroupsv1  #  ...with an optional message
+######################
+function skip_if_rootless_cgroupsv1() {
+    if is_rootless; then
+        if ! is_cgroupsv2; then
+            local msg=$(_add_label_if_missing "$1" "rootless cgroupvs1")
+            skip "${msg:-not applicable under rootless podman}"
```

Comment:
This is misleading; it should include something about cgroupsv1 as well. Perhaps:
```
    ...not supported as rootless under cgroupsv1}"
```